nautobot/dcim/views.py

```diff
@@ -136,6 +136,11 @@ def post(self, request):
         )
 
 
+class BaseDeviceComponentsBulkRenameView(generic.BulkRenameView):
+    def get_selected_objects_parents_name(self, selected_objects):
+        return selected_objects.first().parent.name
```

Comment:
We probably need some error checking and catching here. what if `selected_objects` is `None` or `.parent` is `None`. Give user a message to let them define `selected_objects` in their tests?